src/main/java/com/robertx22/age_of_exile/gui/screens/skill_tree/buttons/drawer/SeparableBufferedImage.java

```diff
@@ -0,0 +1,61 @@
+package com.robertx22.age_of_exile.gui.screens.skill_tree.buttons.drawer;
+
+import com.google.common.collect.ImmutableList;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+public class SeparableBufferedImage {
+
+    private final BufferedImage originalImage;
+    private final int width;
+    private final int height;
+
+    private final int separateTo;
+
+    public SeparableBufferedImage(BufferedImage originalImage) throws IOException {
+        this.originalImage = originalImage;
+        this.width = originalImage.getWidth();
+        this.height = originalImage.getHeight();
+        ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream();
+        ImageIO.write(originalImage, "PNG", byteArrayOutputStream);
+        byte[] bytes = byteArrayOutputStream.toByteArray();
+        byteArrayOutputStream.close();
+        byteArrayOutputStream = null;
+        System.out.println("length is " + bytes.length);
+        this.separateTo = bytes.length / (25 * 1024);
```

Comment:
What are these magic numbers?

Reply:
well Minecraft can only accept images with byte lengths smaller than 64K, so I have to cut it into several parts